apps/kaizen/models.py

```
@@ -28,30 +24,35 @@ class SubTask(BaseModel):
+        (1, "Very Unproductive"),
+        (2, "Unproductive"),
+        (3, "Neutral"),
+        (4, "Productive"),
+        (5, "Very Productive"),
     ]
 
-    date = models.DateField()
     title = models.CharField(max_length=255)
-    reflection = models.TextField()
-    rating = models.IntegerField(choices=RATING_CHOICES)
+    notes = models.TextField(blank=True, default="", null=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)